usmqe_tests/api/tendrl/conftest.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf8 -*-
+
+import pytest
+
+from usmqe.api.tendrlapi.common import TendrlApi
+from usmqe.api.tendrlapi.common import TendrlAuth
+
+
+@pytest.fixture(scope="session")
+def default_session_credentials(request):
```

Comment:
I would rename this fixture to valid_session_credentials so it will be unified with the rest of fixtures. I would also parametrize it in the future so we can test valid user sessions with different roles. (after they will be implemented)